djstripe/models.py

```diff
@@ -497,10 +497,11 @@ def sync_current_subscription(self, cu=None):
             return sub_obj
 
     def update_plan_quantity(self, quantity, charge_immediately=False):
+        sub = self.stripe_customer.subscription
+        if not sub:
+            raise SubscriptionUpdateFailure("Customer does not have a subscription with Stripe")
```

Comment:
What do you think about running ``sync_subscription`` and maybe pointing that out in the exception message? It'll keep the local database updated.